tutorecommerce/patches/openedx-lms-common-settings

```diff
@@ -1,4 +1,4 @@
-ECOMMERCE_PUBLIC_URL_ROOT = "http://ecommerce:8000"
+ECOMMERCE_PUBLIC_URL_ROOT = "http://{{ ECOMMERCE_HOST }}"
```

Comment:
This should probably be: `ECOMMERCE_PUBLIC_URL_ROOT = "{% if ACTIVATE_HTTPS %}https{% else %}http{% endif %}://{{ ECOMMERCE_HOST }}"`